sdk/cosmosdb/cosmos/package.json

```diff
@@ -96,7 +96,8 @@
     "tslib": "^2.2.0",
     "universal-user-agent": "^6.0.0",
     "uuid": "^8.3.0",
-    "@azure/abort-controller": "^1.0.0"
+    "@azure/abort-controller": "^1.0.0",
+    "queue-fifo": "^0.2.6"
```

Comment:
Is this external package still maintained? We could implement a queue on our own.